### PR TITLE
Allows the building of tables in raptors

### LINKED ISSUE #1782: Cannot build tables inside a raptor
Issue reported from  Round ID: 3659 (NSV 13 | Official | MRP | 98% Free Lag | Spaceship Combat Server)

Reporting client version: 514.1575

### PR DESCRIPTION
**Pull request by Pontenerd**
## About The Pull Request
Title
Fixes #1782 

## Why It's Good For The Game
Lets me build a flying bar

## Changelog
:cl:

fix: fixed a bug that prevented tables from being built inside of raptors

/:cl:

### CODE REVIEW

File: nsv13/code/game/general_quarters/dropship.dm [hunk 1]
```diff
@@ -6,7 +6,7 @@
 	icon = 'nsv13/icons/turf/dropship.dmi'
 	icon_state = "brown_fr"
 
-/turf/open/indestructible/dropship
+/turf/open/floor/indestructible/dropship
```
Comment (Bokkiewokkie) [inline nsv13/code/game/general_quarters/dropship.dm:9]: /turf/open/floor/indestructible does not exist, you would have to define it to make this work

**Review by Bokkiewokkie — CHANGES_REQUESTED** [review]

File: nsv13/code/game/general_quarters/dropship.dm (hunk 1 above)
Comment (Pontenerd) [reply nsv13/code/game/general_quarters/dropship.dm:9]: You say this like I have any clue on how to do this

Comment (Pontenerd) [reply nsv13/code/game/general_quarters/dropship.dm]: fixed

File: code/game/turfs/open/_open.dm [hunk 2]
```diff
@@ -45,6 +45,18 @@
 	heavyfootstep = FOOTSTEP_GENERIC_HEAVY
 	tiled_dirt = TRUE
 
+	/turf/open/indestructible/floor
```
Comment (covertcorvid) [inline code/game/turfs/open/_open.dm:48]: Remove the indent on line 48 to fix your linter problem

File: code/game/turfs/open/_open.dm [hunk 3]
```diff
@@ -45,6 +45,18 @@
...
+	icon = 'icons/turf/floors.dmi'
+	icon_state = "floor"
+	footstep = FOOTSTEP_FLOOR
+	barefootstep = FOOTSTEP_HARD_BAREFOOT
+	clawfootstep = FOOTSTEP_HARD_CLAW
+	heavyfootstep = FOOTSTEP_GENERIC_HEAVY
+	tiled_dirt = TRUE
+	explosion_block = 50
+
+
```
Comment (Bokkiewokkie) [inline code/game/turfs/open/_open.dm:59]: Please put this in a seperate file in the nsv13 folder

**Review by Bokkiewokkie — CHANGES_REQUESTED** [review]

File: code/game/turfs/open/_open.dm (hunk 3 above)
Comment (Pontenerd) [reply code/game/turfs/open/_open.dm:59]: image

**Comment by covertcorvid** [comment]
Fixes  #1782

**Comment by covertcorvid** [comment]
These maps are tiny, so if this gets properly modularized I don't mind if we merge it before beebase

**Review by Bokkiewokkie — APPROVED** [review]

**Comment by Pontenerd** [comment]
all my PR with all my code

**Comment by Bokkiewokkie** [comment]
Very PR, much fixes

**Pull request merged**